docs/source/index.rst

```diff
@@ -1,19 +1,43 @@
 .. _simba_ml_docs_mainpage:
 
 Welcome to SimbaML's documentation!
-==============================================
+===================================
 
 **Date**: |today| **Version**: |version|
 
+**SimbaML** is an all-in-one framework for integrating prior knowledge of ODE models into the ML process by synthetic data augmentation.
+It allows for the convenient generation of realistic synthetic data by sparsifying and adding noise.
+Furthermore, our framework provides customizable pipelines for various ML experiments, such as identifying needs for data collection and transfer learning.
+
+.. image:: _static/visualabstract.png
+  :width: 2000
+  :alt: Visual Abstract of SimbaML
+
 **Useful links**:
 :ref:`installation` |
 `Source Repository <https://github.com/DILiS-lab/SimbaML>`_ |
 `Issue Tracker <https://github.com/DILiS-lab/SimbaML/issues>`_ |
 `Mailing List <mpws2022br1@hpi.de>`_
 
-**SimbaML** is a tool for generating time-series data by using ordinary differential equations.
-After generating data, multiple ML-approaches can be evaluated and compared.
+Installation
+------------
 
-.. image:: _static/visualabstract.png
-  :width: 2000
-  :alt: Visual Abstract of SimbaML
+SimbaML requires Python 3.10 or newer and can be installed via pip:
+
+..  code-block:: bash
+    pip install simba_ml
+
+You can check if the installation was successful by importing the package and checking the version:
+
+.. code-block:: python
+    import simba_ml
+    simba_ml.__version__
+
+For more detailed installation instructions and requirements, see :ref:`installation`.
+
+For an example usage of SimbaML, see :ref:`quickstart`.
+
+Links
+-----
+
+See our ICLR Tiny Paper: https://arxiv.org/pdf/2304.04000.pdf
```

Comment:
This links to Arxiv and not to ICLR.

Reply:
Very true, the ICLR version is still in the making. So this is a tempary link. Should I put "on Arxiv" at the end of the sentence?

Reply:
See cite us:
https://github.com/DILiS-lab/SimbaML/blob/main/docs/source/about/cite.rst

Reply:
By the way: If you want, you can change Submitted to Accepted in that file.

Reply:
Should be fixed in the latest commit.